### ISSUE TITLE
iOS Returns True for Compatibility

### ISSUE THREAD

**Issue by roblav96** [NathanaelA/nativescript-permissions#10]
I was thinking, for cross-platform compatibility, instead of

``` javascript
if (!global.android) {
    return;
}
```

maybe just return true for iOS?

``` javascript
function hasPermission(perm) {
    if (!hasSupportVersion4()) return true;
    if (!global.android) return true;

    var hasPermission = android.content.pm.PackageManager.PERMISSION_GRANTED ==
        android.support.v4.content.ContextCompat.checkSelfPermission(application.android.foregroundActivity, perm);
    return (hasPermission);
}
```

Thoughts?

I just saw this issue,  https://github.com/NathanaelA/nativescript-permissions/issues/7

**Comment by NathanaelA** [comment]
I just published a new version (v1.2.0) that has a ios compatibility shim which will return "true" to any and all permissions asked...  ;-)

**Comment by roblav96** [comment]
Perfect!!! much <3 :D
